src/NCalc/Domain/EvaluationVisitor.cs

```diff
@@ -276,333 +284,186 @@ public override void Visit(Function function)
             return;
         }
 
-        switch (function.Identifier.Name.ToLower())
-        {
-            #region Abs
-            case string n when n.Equals("abs", StringComparison.OrdinalIgnoreCase):
+        ExecuteBuiltInFunction(function);
+    }
 
-                CheckCase("Abs", function.Identifier.Name);
+    private void ExecuteBuiltInFunction(Function function)
+    {
+        var functionName = function.Identifier.Name.ToLower();

```

Comment:
It'll be better to replace `ToLower()` with `ToUpperInvariant()`
You can find more discussions here: https://github.com/sklose/NCalc2/pull/84